src/handlers/cross-account-transfer/handler.ts

```diff
@@ -0,0 +1,105 @@
+import { SQSClient, SendMessageBatchCommand } from '@aws-sdk/client-sqs';
+import { S3Client, ListObjectsV2Command, GetObjectCommand } from '@aws-sdk/client-s3';
+import { createGunzip } from 'zlib';
+import { v4 as uuidv4 } from 'uuid';
+
+const sqs = new SQSClient({ apiVersion: '2012-11-05', region: 'eu-west-2' });
+const s3 = new S3Client({ region: 'eu-west-2' });
```

Comment:
minor point but we define all these clients once in the file `shared/clients.ts` so you can just import them from there

Reply:
Thanks for the observation. Used the client defined in the `shared/client.ts` and appropriately import them.